src/Runtime/python/onnxmlir/pyproject.toml

```diff
@@ -16,11 +16,15 @@ classifiers = [
     "License :: OSI Approved :: Apache Software License",
     "Operating System :: POSIX :: Linux",
 ]
-dependences = [
-    "numpy",
-    "docker",
+
```

Comment:
Somewhere there should be something that allow the install package to say "sorry, not supported on platform X". Ok to be zLinux only for now.